assets/prototype/domain/eventEditor/DatesList.js

```diff
@@ -0,0 +1,18 @@
+import { useContext } from '@wordpress/element';
+import { DatesListDataContext } from './containers/datesListData/index';
+
+const DatesList = () => {
+	const list = useContext( DatesListDataContext );
+
+	return list ? (
+		<ul>
+			{ list.map( ( { id, name } ) => (
+				<li key={ id }>
+					{ name }
+				</li>
+			) ) }
```

Comment:
Here we should use an individual component to render a single entity and pass the ID to that component. Then that component will get its data from cache using that ID.